journal/src/main/scala/com/evolutiongaming/kafka/journal/Journal.scala

```diff
@@ -332,17 +332,25 @@ object Journal {
     implicit val configReaderConsumerPoolConfig: ConfigReader[ConsumerPoolConfig] = deriveReader
   }
 
-  trait DataIntegrityConfig {
-    def seqNrUniqueness: Boolean
-  }
+  final case class DataIntegrityConfig(
+    /**
+      * If true then duplicated [[SeqNr]] in events will cause [[JournalError]] `Data integrity violated`
+      */
+    seqNrUniqueness: Boolean,
+
+    /**
+      * If true then events with [[CorrelationId]] different from one in metadata will be filtered out
```

Comment:
I think you missed this one in refactoring. I'd suggest to use text search on the project just to be sure.

Reply:
fixed, thanks